src/index.html

```diff
@@ -13,6 +13,44 @@
     />
```

Comment:
The stylesheet link should reference a CSS file, not a SCSS file. SCSS files need to be compiled to CSS before they can be used in HTML.